### PR TITLE
[ENH] create lists for language, input type, data type for converters tables

### LINKED ISSUE #248: Update  date of last commit for each converter
Random thought 

- Write python script to ping github api for last commit date of each converter to update yaml file

- add monthly workflow to run script and open pr in case of changes.

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #248

### CODE REVIEW

File: _data/MEEG_converters.yml [hunk 1]
```diff
@@ -19,21 +22,32 @@
...
       updated: 2021-08-18
       license: GPL-2.0
 
     - name: MNE-BIDS
       url: https://mne.tools/mne-bids/
-      data_types: MEG, EEG, iEEG
+      data_types: 
+        - MEG
+        - EEG
+        - iEEG
```
Comment (sappelhoff) [inline _data/MEEG_converters.yml:44]:
```suggestion
        - iEEG
        - NIRS
```

**Comment by sappelhoff** [comment]
it's funny: clicking on the netlify deploy preview works pretty well --> UNTIL you click on the "benefits" tab ... which redirects you from the deploy PREVIEW to the actual deployment :confused: can you replicate this @Remi-Gau?

**Comment by Remi-Gau** [comment]
> it's funny: clicking on the netlify deploy preview works pretty well --> UNTIL you click on the "benefits" tab ... which redirects you from the deploy PREVIEW to the actual deployment confused can you replicate this @Remi-Gau?

Yup I had seen that before even locally sometimes (I think).

**Comment by Remi-Gau** [comment]
fyi the actual page would be this one:

https://deploy-preview-261--astounding-beignet-8ca2d6.netlify.app/benefits.html#mri-and-pet-converters

**Comment by Remi-Gau** [comment]
seems to only affect the dropdown menu of the navbar

not sure why

**Review by sappelhoff — APPROVED** [review]

**Comment by effigies** [comment]
This table is extremely wide. On my laptop, the left/right borders are off the edge of my scrollable screen. On a large desktop monitor I can see everything.

I think we need something more compact.

**Comment by Remi-Gau** [comment]
agreed this has been bugging me for a while actually

I have started playing with the CSS but not found something that would make it more manageable / readable if we keep some much info into the table.

**Comment by effigies** [comment]
I think there might be a solution with colored tags/badges and a legend. Modality, language, GUI, docs, license could all be represented very compactly.

**Comment by Remi-Gau** [comment]
@effigies 

https://deploy-preview-261--astounding-beignet-8ca2d6.netlify.app/benefits.html#mri-and-pet-converters

how does that look to you?

**Comment by effigies** [comment]
Definitely more reasonably sized. I would also change the pypi and docker links to point to the projects on pypi and docker. We could also turn them into shields. Maybe it would make sense to just directly put the shield for the project into the YAML rather than rendering it on the fly with Jinja.

**Comment by Remi-Gau** [comment]
> Definitely more reasonably sized. I would also change the pypi and docker links to point to the projects on pypi and docker. We could also turn them into shields. 

Will look into that.

> Maybe it would make sense to just directly put the shield for the project into the YAML rather than rendering it on the fly with Jinja.

Not sure. Given that we use the repos url for other things (want to see if we can "easily" generate a page for each tool where all the info for this tool is displayed). So would not make that change just yet.

**Comment by Remi-Gau** [comment]
@effigies @sappelhoff 

now with dockerhub, pypi, conda-forge badges

https://deploy-preview-261--astounding-beignet-8ca2d6.netlify.app/benefits.html#mri-and-pet-converters

**Comment by sappelhoff** [comment]
nice! However, the conda badge doesn't say "conda", but when that conda package was "last updated", which is not the important information that users would need:

image

**Comment by Remi-Gau** [comment]
> nice! However, the conda badge doesn't say "conda", but when that conda package was "last updated", which is not the important information that users would need:
> 
> image

The badge should now display the version on anaconda.org.

**Review by sappelhoff — APPROVED** [review]

**Pull request merged**